consensus/src/round_manager.rs

```diff
@@ -1197,6 +1253,9 @@ impl RoundManager {
                 state = vote.vote_data().proposed().executed_state_id(),
                 is_timeout = vote.is_timeout(),
             );
+            if self.local_config.enable_round_timeout_msg {
+                error!("Received Vote with Timeout with Round Timeout Message enabled.");
```

Comment:
I don't think this holds, during rollout, some nodes will have the flag enabled while others don't. the flag can only control sender side but not receiver

Reply:
but it'll happen all the time during rollout (if there's timeout)

Reply:
Yes, that is what my expectation is. If it can be useful to debug, why not? Don't you think it's useful? I can downgrade to warn as well.

Reply:
we probably don't want noise during the rollout, this doesn't give much information as well since it's expected during the release

Reply:
Removed it. Done.